doc/reference/storage_drivers.md

```diff
@@ -67,9 +69,15 @@ Btrfs, ZFS and Ceph RBD have an internal send/receive mechanism that allows for
 LXD uses this optimized transfer when transferring instances and snapshots between storage pools that use the same storage driver, if the storage driver supports optimized transfer and the optimized transfer is actually quicker.
 Otherwise, LXD uses `rsync` to transfer container and file system volumes, or raw block transfer to transfer virtual machine and custom block volumes.
 
-The optimized transfer uses the underlying storage driver's native functionality for transferring data, which is usually faster than using `rsync`.
-However, the full potential of the optimized transfer becomes apparent when refreshing a copy of an instance or custom volume that uses periodic snapshots.
-With optimized transfer, LXD bases the refresh on the latest snapshot, which means:
+The optimized transfer uses the underlying storage driver's native functionality for transferring data, which is usually faster than using `rsync` or raw block transfer.
+
+(storage-optimized-volume-refresh)=
+### Optimized volume refresh
+
+The full potential of the optimized transfer becomes apparent when refreshing a copy of an instance or custom volume that uses periodic snapshots.
+If the optimized transfer isn't supported by the driver or refresh, instead of the delta, the entire volume including its snapshot(s) will be copied using either `rsync` or raw block transfer. LXD will try to keep the overhead low by transferring only the volume itself or any snapshots that are missing on the target.
```

Comment:
> If the optimized transfer isn't supported by the driver **or refresh**

What do you mean by the optimized transfer not being supported by refresh? Do you mean "If the optimized transfer or the optimized refresh isn't supported by the driver"?

Reply:
All drivers except Ceph RBD that have support for the optimized volume transfer also have support for optimized volume refresh. I tried to explain that "optimized volume transfer" doesn't necessarily mean you automatically get "optimized volume refresh".

Maybe it can be rephrased to this:

"If the optimized transfer isn't supported by the driver or its implementation of volume refresh, ..."